app/models/order.rb

```diff
@@ -213,8 +214,12 @@ def send_new_order_notificationen
     }
   end
 
-  def send_new_order_sms
-    TwilioService.new(submitted_by).send_new_order_sms(self)
+  def send_new_order_confirmed_sms_to_charity
+    TwilioService.new(submitted_by).order_confirmed_sms_to_charity(self)
```

Comment:
Can we pass order object to initializer instead of argument to method ?

Reply:
We can but what difference will it make?? I mean whats the advantage of passing it to initializer instead of passing to method?
I only need to use it in one method so I think passing to initializer would be more useful if it's needed in the Class somewhere else too.

What do you think @namrataukirde ?

Reply:
Ohh okay, So we are using it in single method. Then approach you have used is correct. Thanks for clarification @Darshan-Chauhan :)

Reply:
Sure :)